demo/tekton-demo-setup/03-build-and-deploy.yaml

```diff
@@ -93,8 +93,13 @@ objects:
           script: |
             set -ex
             find /workspace
-            cd /workspace/debug 
-            git log -1 --format=%cd > "$(results.commit-time.path)"
+            cd /workspace/debug
+            printf "epoch timestamps are recommended: git log -1 --format=%ct"
```

Comment:
These printf calls won't have newlines, any problem with `echo`?

Also, what if `%c` is interpreted by printf? Could be brittle.

Reply:
should be:

```
printf "epoch timestamps are recommended: git log -1 --format=%%ct\n"
```

or as @KevinMGranger suggested:
```
echo "epoch timestamps are recommended: git log -1 --format=%ct"
```

Reply:
it works and looks fine imho, here's the output.
```
+ cd /workspace/debug
+ printf 'epoch timestamps are recommended: git log -1 --format=%ct'
epoch timestamps are recommended: git log -1 --format=t++ git log -1 --format=%ct
+ epoch=1663770043
++ date -d @1663770043
+ human='Wed Sep 21 14:20:43 UTC 2022'
epoch time: + printf 'epoch time: ' + 1663770043
human readable time: + printf 'human readable time: ' + Wed Sep 21 14:20:43 UTC 2022
+ echo 1663770043
STEP-DEBUG-TIME-LAST-COMMIT

1663770043
```

Reply:
`epoch timestamps are recommended: git log -1 --format=t`
It swallowed the `%c`. You only see the correct output because it's in debug mode where it echoes each line (`set -x`).

Since it's just for debugging I guess it's fine, but this is brittle.